packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -91,14 +91,22 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
             resolvedUrl.endpoints.ordererUrl,
         );
         // the backend responds with the actual document ID associated with the new container.
-        const documentId = await ordererRestWrapper.post<string>(
+        const { id: documentId, token } = await ordererRestWrapper.post<{id: string, token: string}>(
```

Comment:
These driver changes would not be released at the same time as the service changes, so token should be optional here, and `token===undefined` should be handled accordingly

Reply:
Does it mean it need to handle temporary two return types `string` and `{id: string, token: string}`, until the service is deployed?

Reply:
Correct, yes. I believe you could check this with `if (typeof response === "string")`